Hardware Info.py

```diff
@@ -1,9 +1,30 @@
 import cpuinfo
 from tkinter import *
 from tkinter import ttk
+import psutil
+from itertools import count
+import matplotlib.pyplot as plt
+from matplotlib.animation import FuncAnimation
 
 
 class Hardware:
+    def cpu_load_start(self):
+        plt.style.use("fivethirtyeight")
+        time = []
+        load_val = []
+        time_sec = count()
+
+        def cpu_load(i):
+            time.append(next(time_sec))
+            load_val.append(psutil.cpu_percent(interval=0))
+            plt.cla()
+            plt.plot(time, load_val, label="CPU Load in % over time (s)")
+            plt.legend(loc="upper left")
+            plt.tight_layout()

```

Comment:
This function will be called every second using FuncAnimation provided by matplotlib to plot the cpu load values that updates every second